draft-ietf-rtcweb-jsep.xml

```diff
@@ -1885,15 +1885,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           session-level attributes. The process here is identical to that
           indicated in the Initial Offers section above.</t>
 
-	  <t>The next step is to generate lip sync groups as defined
-	  in <xref target="RFC5888"/>, Section 7.  For each
-	  MediaStream with more than one MediaStreamTrack, a group
-	  of type "LS" MUST be added that contains the mid values for
-	  each MediaStreamTrack in that MediaStream. In some cases
-	  this may result in adding a mid to a given LS group that
-	  was not in that LS group in the associated offer. Although
-	  this is not allowed by <xref target="RFC5888"/>, it is
-	  allowed when implementing this specification. </t>
+	  <t>The next step is to generate session-level lip sync
+	  groups as defined in <xref target="RFC5888"/>, Section
+	  7. All Tracks in the same MediaStream need to be placed in
+	  the same "LS" group as long as theses tracks were in the
+	  offered to by synchronized in the offer.
```

Comment:
Suggest alternate text:

If all tracks of a MediaStream that the answerer wants to send fit within the media sections of an LS group in the offer, these media sections (and only these media sections) are used for the MediaStream. If there are media sections in the offer that are not part of an LS group, these can be used by any track belonging to any stream.
In all cases, the LS groups in the answer are identical to the LS groups in the offer.